packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -84,6 +84,9 @@ class WalletsController {
         addresses: key.addresses!,
       })
       windowManage.broadcast(Channel.Wallets, WalletsMethod.GetAll, WalletsController.getAll())
+      if (WalletsController.service.getAll().length === 1) {
```

Comment:
the broadcast of active wallet should be invoked when the active wallet updated, nothing related to the import mnemonic

Reply:
I've added active wallet verification.There are two places to set active wallet.So I broadcast in these places.If I broadcast in service with windowManager,the wallet store service test and address service test should add 'electon-enviroment' to test.It will make things complicated.